pkg/alertmanager/alertmanager_metrics_test.go

```diff
@@ -727,6 +767,27 @@ func TestAlertmanagerMetricsRemoval(t *testing.T) {
 			# HELP cortex_alertmanager_silences_snapshot_size_bytes Size of the last silence snapshot in bytes.
 			# TYPE cortex_alertmanager_silences_snapshot_size_bytes gauge
 			cortex_alertmanager_silences_snapshot_size_bytes 11
+
+		# HELP cortex_alertmanager_state_fetch_replica_state_failed_total Number of times we have failed to read and merge the full state from another replica.
```

Comment:
Wrong indentation.

Reply:
This whole file has bizarre indentation - the string contents are a mix of tabs and spaces. I presume you want six tabs?

Reply:
I think the number of tabs is not that important as far as they're consistent ;)